### PR TITLE
[BD-21] Upgrade usage of edx_toggles.toggles

### PR DESCRIPTION
**Pull request by regisb**
**TL;DR -** 

The waffle classes previously in __future__ are now available directly from
edx_toggles.toggles. Since this __future__ module is going to be removed soon,
we become future-proof by getting rid of it.

JIRA: [JIRA-BD-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation)

**Developer Checklist**

- [x] Reviewed the [release process](./release_process.md)
- [ ] ~~Translations and JS/SASS compiled~~
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

FYI: @edx/masters-devs-gta

Note that the package version was obsolete in package.json.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @regisb! I've created [BLENDED-761](https://openedx.atlassian.net/browse/BLENDED-761) to keep track of it in Jira.   More details are on the [BD-21](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1596358943/BD-21+Toggles+Settings+Documentation) project page.


When this pull request is ready, tag your edX technical lead.

**Review by robrap — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "2.13.9",
+  "version": "3.0.2",
```
Comment (robrap) [inline package.json:3]: @mattcarter: FYI: Should there be a comment or PR template item for keeping this in sync with the setup.py version?
